packages/trilinoscouplings/examples/scaling/TrilinosCouplings_Statistics.hpp

```diff
@@ -252,14 +252,13 @@ class MachineLearningStatistics_Hex3D {
       double ratio = edge_length_1 / edge_length_2;
       ratio = std::min(ratio, 1/ratio);
       for (int j=0; j<NUM_EDGE_PAIRS; j++) {
-	node1 = elemToNode(i, edge_nodes_1[edge_opposites_1[j]]);
-	node2 = elemToNode(i, edge_nodes_2[edge_opposites_1[j]]);
-	node3 = elemToNode(i, edge_nodes_1[edge_opposites_2[j]]);
-	node4 = elemToNode(i, edge_nodes_2[edge_opposites_2[j]]);
-	edge_length_1 = distance2(nodeCoord, node1, node2);
-	edge_length_2 = distance2(nodeCoord, node3, node4);
-	double ratio = edge_length_1 / edge_length_2;
-	ratio = std::min(ratio, 1/ratio);
+        node1 = elemToNode(i, edge_nodes_1[edge_opposites_1[j]]);
+        node2 = elemToNode(i, edge_nodes_2[edge_opposites_1[j]]);
+        node3 = elemToNode(i, edge_nodes_1[edge_opposites_2[j]]);
+        node4 = elemToNode(i, edge_nodes_2[edge_opposites_2[j]]);
+        edge_length_1 = distance2(nodeCoord, node1, node2);
+        edge_length_2 = distance2(nodeCoord, node3, node4);
+        ratio = std::min(ratio, 1/ratio);
```

Comment:
Yeah, this ain't right.

Reply:
The question is, what did you intend to happen?

Reply:
Good question

Reply:
Taper: Maximum ratio of lengths derived from opposite edges.